ember-addon-main.js

```diff
@@ -18,26 +18,26 @@ module.exports = {
 
   parentRegistry: null,
 
-  shouldSetupRegistryInIncluded: function() {
+  shouldSetupRegistryInIncluded() {
```

Comment:
This should go, we should remove support for old ember-cli during the major bump